interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -678,6 +694,32 @@ protected ManagedChannelBuilder<?> createChannelBuilder() {
         if (addMdInterceptor != null) {
           nettyBuilder.intercept(addMdInterceptor);
         }
+        if (useOpenTelemetryTracing) {
+          OpenTelemetrySdk openTelemetrySdk = AutoConfiguredOpenTelemetrySdk.builder()
+              .addPropagatorCustomizer((reader, config) -> {
+                if (config.getString("otel.propagators") == null) {
+                  return GrpcTraceBinContextPropagator.defaultInstance();
+                } else {
+                  return reader;
+                }
+              })
+              .addPropertiesSupplier(() -> ImmutableMap.of(
+                  "otel.logs.exporter", "none",
+                  "otel.traces.exporter", "none",
+                  "otel.metrics.exporter", "none"))
+              .build()
+              .getOpenTelemetrySdk();
+
+          GrpcOpenTelemetry.Builder grpcOpentelemetryBuilder = GrpcOpenTelemetry.newBuilder()
+              .sdk(openTelemetrySdk);
+          InternalGrpcOpenTelemetry.enableTracing(grpcOpentelemetryBuilder, true);
+          GrpcOpenTelemetry grpcOpenTelemetry = grpcOpentelemetryBuilder.build();
+          // Disabling census-tracing is necessary to avoid trace ID mismatches.
```

Comment:
The auto-applying is still causing trouble... This is a symptom of larger issues. It's okay for a while, but our users may have to deal with the same.